Add `vault_pki.read_certificate_full` (#146)
salt-extensions/saltext-vault: changelog/145.added.md

> Add `read_certificate_full` execution function to the `vault_pki` module, returning the certificate, private key, and CA chain as a dictionary instead of a plain string.

Reviewer: question: The private key is not part of the response, right? It's an unauthenticated endpoint.

```suggestion
Added `read_certificate_full` execution function to the `vault_pki` module, returning the certificate, its CA chain and miscellaneous information as a dictionary.
```
